Userland/Libraries/LibWeb/URL/URLSearchParams.cpp

```diff
@@ -149,6 +149,13 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<URLSearchParams>> URLSearchParams::construc
     return URLSearchParams::create(realm, url_decode(stripped_init));
 }
 
+// https://url.spec.whatwg.org/#dom-urlsearchparams-size
+size_t size() const
```

Comment:
This doesn't build, you have to specify the class name: `size_t URLSearchParams::size() const`

Reply:
Oops, sorry, fixed.